auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -34,25 +34,27 @@ class CheckPullRequest extends AuthenticatedRequestHandler {
   final ApproverServiceProvider approverProvider;
 
   static const int kPullMesssageBatchSize = 100;
+  static const int kPubsubPullNumber = 5;
```

Comment:
How many iterations can we run in 60s? I recommend making this # closer to a 1/2 of the max.

Reply:
I need to monitor/adjust this a bit based on the response time. Here 5 is a relatively conservative number to begin with.

Reply:
Please add a TODO to not forget to update this number to an optimal value.

Reply:
Added